dotnet/.ci/dotnet-hello-world.cloudbuild.yaml

```diff
@@ -0,0 +1,75 @@
+steps:
+  - id: Create namespace
+    name: 'gcr.io/cloud-builders/kubectl'
+    entrypoint: '/bin/bash'
+    args:
+    - '-c'
+    - |
+      kubectl config view
+      gcloud container clusters get-credentials $$CLOUDSDK_CONTAINER_CLUSTER
+      kubectl config view
+      kubectl create namespace $$TEST_NAMESPACE
+      echo "Kubernetes Namespace is $$TEST_NAMESPACE"
+    dir: '${_SAMPLE_DIR}'
```

Comment:
_SAMPLE_DIR is not defined later.
You either need to define it later (end of file) or use inline the full address of the folder (in the next step there is an example).